source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -65,6 +70,22 @@ async def fetch_properties(
     return properties_cache[object_name]
 
 
+async def fetch_deal_pipelines(
+    log: Logger, http: HTTPSession
+) -> DealPipelines:
+    url = f"{HUB}/crm-pipelines/v1/pipelines/deals"
```

Comment:
For my own understanding, was the motivation to use [`/crm-pipelines/v1/pipelines/deals`](https://developers.hubspot.com/beta-docs/reference/api/crm/pipelines/v1) instead of [`/crm/v3/pipelines/deals`](https://developers.hubspot.com/beta-docs/reference/api/crm/pipelines#get-%2Fcrm%2Fv3%2Fpipelines%2F%7Bobjecttype%7D) because the required permissions for the former are simpler?

Reply:
I didn't really think about it too much. I think they are basically equivalent. The v1 APIs tend to have more lenient rate limits.